dev/com.ibm.ws.jsp/bnd.bnd

```diff
@@ -116,11 +108,34 @@ Import-Package: \
    org.eclipse.jdt.internal.compiler.problem;version="3.10.2";usage="JSP", \
    sun.io;resolution:=optional, \
    *
-   
+
+DynamicImport-Package:\
+  org.apache.taglibs.standard,\
+  org.apache.taglibs.standard.extra.spath,\
+  org.apache.taglibs.standard.functions,\
+  org.apache.taglibs.standard.lang.jstl,\
+  org.apache.taglibs.standard.lang.jstl.parser,\
+  org.apache.taglibs.standard.lang.jstl.test,,\
+  org.apache.taglibs.standard.lang.jstl.test.beans,\
+  org.apache.taglibs.standard.lang.support,\
+  org.apache.taglibs.standard.resources,\,\
+  org.apache.taglibs.standard.tag.common.core,\
+  org.apache.taglibs.standard.tag.common.fmt,\
+  org.apache.taglibs.standard.tag.common.sql,\
+  org.apache.taglibs.standard.tag.common.xml,\
+  org.apache.taglibs.standard.tag.el.core,\
+  org.apache.taglibs.standard.tag.el.fmt,\
+  org.apache.taglibs.standard.tag.el.sql,\
+  org.apache.taglibs.standard.tag.el.xml,\
+  org.apache.taglibs.standard.tag.rt.core,\
+  org.apache.taglibs.standard.tag.rt.fmt,\
+  org.apache.taglibs.standard.tag.rt.sql,\
+  org.apache.taglibs.standard.tag.rt.xml,\
+  org.apache.taglibs.standard.tei,\
+  org.apache.taglibs.standard.tlv
+
```

Comment:
Why are these dynamic imports?

Reply:
I can see that you might want a version range on the import but not sure why that would require them to be dynamic?

Reply:
When I tried adding these imports in the standard import, they weren't properly included in the manfiest.mf. That let to unresolved bundles during startup.  

But will take another look at this because I don't like the dynamic imports either.